packages/api/lambdas/queue.js

```diff
@@ -1,34 +0,0 @@
-'use strict';
```

Comment:
Nice finding some dead code and removing it! 💯

Reply:
Well Alireza confirmed it was dead and Marc kinda found it, so I can only take credit for the actual physical removal.